src/main/java/HomeWork5/findUnique.java

```diff
@@ -0,0 +1,35 @@
+package HomeWork5;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class findUnique {
+    public static int singleElement(int[] arr)
```

Comment:
You should work with List 

To you coming list and in result list, but without repeats values (any only one time)